### ISSUE TITLE
Android: transparent strip on dirt_with_grass

### ISSUE THREAD

**Issue by Wayward1** [freeminer/freeminer#206]
I've noticed this for a while now. There is a transparent strip between the dirt and grass pixels, which allows the player to see underground as if using noclip.

**Comment by C1ffisme** [comment]
Weird.

Do you have any kind of Shaders on?

**Comment by Wayward1** [comment]
Nope. Upon further testing, it seems to occur only with mipmapping enabled.

**Comment by Wayward1** [comment]
Fwiw, it happens with different texture packs too. Also, it may be worth noting that this doesn't happen in the Minetest app.

**Comment by Wayward1** [comment]
Just recently I noticed this also happens when digging a node. Certain pixels become transparent when the cracks appear.

**Comment by Wayward1** [comment]
Water too:
![screenshot_2015-04-22-22-41-55](https://cloud.githubusercontent.com/assets/8821916/7335771/0345fc2c-eba3-11e4-93a9-a31881a38725.png)

**Comment by C1ffisme** [comment]
Hmmm... Could you describe your Android's system stuff? Like video card?

**Comment by Wayward1** [comment]
http://www.cnet.com/products/amazon-fire-hd-6/

**Comment by C1ffisme** [comment]
Hmm... Not really, I was hoping for hardware specifications (Other than cameras and sensors).

**Comment by Wayward1** [comment]
CPU: Quad-core MediaTek MT8135 1.5 GHz
GPU: Imagination Technologies PowerVR Series6
RAM: 1GB

**Comment by C1ffisme** [comment]
Just a question, does the bug still apear if you turn fancy leaves off?

**Comment by Wayward1** [comment]
I never have them on. I just tried with them on, same result.

**Comment by proller** [comment]
fixed?
